tests/api/test_web_routes_task_shapes.py

```
@@ -351,3 +351,441 @@ async def test_execute_task_from_project_preserves_idle_no_ready_work_truth():
+            create_project_async=AsyncMock(
+                side_effect=ValidationError("Project 'Demo' already exists", field="name")
+            ),
+        )
+    )
+
+    with pytest.raises(HTTPException) as exc_info:
+        await initialize_project_from_blueprint(
+            core=core,
+            name="Demo",
+            description=None,
+            workspace_path="/tmp/demo",
```

Ruff (0.15.11)

[error] Probable insecure usage of temporary file or directory: "/tmp/demo"

(S108)